mailpoet/assets/js/src/global.d.ts

```diff
@@ -90,4 +90,6 @@ interface Window {
   mailpoet_premium_plugin_download_url: string;
   mailpoet_premium_plugin_activation_url: string;
   mailpoet_plugin_partial_key: string;
+  mailpoet_email_volume_limit: string;
+  mailpoet_email_volume_limit_reached: string;
```

Comment:
The variable name sounds like this one should be boolean. Is the type correct?

Reply:
True, I will fix it.